### ISSUE TITLE
When submitting Google ReCaptcha keys, the form fields become hidden. 

### ISSUE THREAD

**Issue by daamsie** [Charitable/Charitable-Spam-Blocker#6]
![Screen Shot 2020-12-14 at 11 59 45 am](https://user-images.githubusercontent.com/740324/102030251-1ea83c80-3e06-11eb-9e46-9b09d1023af2.png)

**Comment by ericnicolaas** [comment]
Yeah, this is a bug in Charitable, with the way it handles these conditional fields. I'd be keen to have you take a look at how to do this better, because the approach I'm taking here often seems to cause issues like this.
